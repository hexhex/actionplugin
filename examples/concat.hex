stringpair("ab","cd").

fullstring(Full) :- &concat[P,F](Full), stringpair(P,F).
