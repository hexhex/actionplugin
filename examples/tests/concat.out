{fullstring("abcd"), stringpair("ab","cd")}
